index.html

```diff
@@ -8,49 +9,254 @@
     <link rel="stylesheet" href="style.css">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 </head>
+
 <body>
-    
+
     <header>
-        <a href="#" class="logo"> <./ Vi$hal> </a>
-       
+        <a href="#" class="logo">
+            <. / Vi$hal>
+        </a>
+
         <nav class="burger">
             <i class="fa fa-bars fa-2x" aria-hidden="true"></i>
         </nav>
     </header>
-        <section class="content">
-            <div class="introduction">
-                <h1 class="intro">I’m Vishal. <br> Glad to see <br> you!</h1>
-                <p class="description">I’m a software developer! I can help you build a product , feature or website Look
-                    through some of my work and experience! If you like what you see and have a project you need coded,
-                    don’t hestiate tocontact me.</p>
+    <section class="content">
+        <div class="introduction">
+            <h1 class="intro">I’m Vishal. <br> Glad to see <br> you!</h1>
+            <p class="description">I’m a software developer! I can help you build a product , feature or website Look
+                through some of my work and experience! If you like what you see and have a project you need coded,
+                don’t hestiate tocontact me.</p>
+        </div>
+
+        <div class="social-media">
+            <a href="#" class="contact">LET'S CONNECT </a>
+            <ul class="social-link">
+                <li class="tweet"><a href="#">
+                        <i class="fa fa-twitter fa-2x" aria-hidden="true"></i>
+                    </a>
+                </li>
+                <li><a href="#">
+                        <i class="fa fa-linkedin fa-2x" aria-hidden="true"></i>
+                    </a>
+                </li>
+                <li><a href="#">
+                        <i class="fa fa-medium fa-2x" aria-hidden="true"></i>
+                    </a>
+                </li>
+                <li><a href="#">
+                        <i class="fa fa-github fa-2x" aria-hidden="true"></i>
+                    </a>
+                <li><a href="#">
+                        <i class="fa fa-angellist fa-2x" aria-hidden="true"></i>
+                    </a>
+                </li>
+            </ul>
+        </div>
+
+    </section>
+
+    <section class="Projects">
```

Comment:
- [ ] Please I suggest displaying this section with a CSS grid. 😃

Reply:
Well done overall on the project 💯 but as requested by the previous reviewer the main container of the projects should have a `display:grid` 🧉 this really helps with the responsiveness 📱